aip/general/0158.md

```diff
@@ -92,6 +92,26 @@ message ListBooksResponse {
   - This total **may** be an estimate (but the API **should** explicitly
     document that).
 
+### Skipping results
+
+The request definition for a paginated operation **may** define an `int32 skip`
+field to allow the user to skip results.
+
+The `skip` value **must** refer to the number of individual resources to skip,
+not the number of pages.
+
+For example:
+
+- A request with no page token and a `skip` value of `30` returns a single page
+  of results starting with the 31st result.
+- A request with a page token corresponding to the 51st result (because the
+  first 50 results were returned on the first page) and a `skip` value of `30`
+  returns a single page of results starting with the 81st result.
```

Comment:
Can you add an example with negative `skip`?

Reply:
I want to intentionally not discuss negative `skip` for now. The working group was not sure if negative `skip` was wise and I am not yet ready to either suggest it or prohibit it. I am pretty willing for services to make their own decisions.

Reply:
Actually, changed my mind. I will mention it briefly.

Reply:
Nope, changed my mind again. Leaving as-is for now. I want to bring the question to others first.